recipes/legit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set packagename = 'legit' %}
+{% set version = "0.2.0" %}
+{% set md5 = "5df3cd402b0de8f0251a87904b5eb88c" %}
+
+package:
+  name: {{ packagename }}
+  version: {{ version }}
+
+source:
+  fn: {{ packagename }}-{{ version }}.tar.gz
```

Comment:
change your filename to end with ".zip" since you're downloading a zipball from pypi, not a tarball